modules/data.atmosphere/R/download.raw.met.module.R

```diff
@@ -0,0 +1,64 @@
+.download.raw.met.module <- function(dir, met, register, machine, start_date, end_date, con, new.site=NULL, host, overwrite=FALSE) {
```

Comment:
This one, along with the other *.module files, simply moved big chunks of code out of met.process. I kind of had @mdietze's #1009 idea in mind, but this is as far as I got. In any case, it's definitely saved my scroll-wheel finger some miles.